src/main/resources/application.conf

```diff
@@ -38,110 +46,224 @@ etc-client {
         "enode://fba5a07e283d517a2680bcfc7aeb498ac2d246d756556a2ebd5edeb39496491c47a6d27e27f82833b7d7d12defc8de994de04bb58beb72472649f9a323006820@41.135.121.6:30303"
       ]
 
+      # Time interval for scanning bootstrap nodes
+      # TODO: explain better
       bootstrap-nodes-scan-interval = 2 minutes
     }
 
     peer {
+      # Retry delay for failed attempt at connecting to a peer
       connect-retry-delay = 20 seconds
+
+      # Maximum number of attempts to connect to a peer
+      # TODO: what happens after that? blacklisting?
       connect-max-retries = 30
+
+      # TODO: what's this for? why delay a PoisonPill
```

Comment:
Discussed with @mcsherrylabs, I will be removing the docs for such timeouts.

But I'm still curious, why do we want to delay the PoisonPill?